src/wasm-lib/kcl/src/execution/state.rs

```diff
@@ -34,6 +33,8 @@ pub(super) struct GlobalState {
     pub id_generator: IdGenerator,
     /// Map from source file absolute path to module ID.
     pub path_to_source_id: IndexMap<ModulePath, ModuleId>,
+    /// Map from module ID to source file.
+    pub path_to_source: IndexMap<ModuleId, ModuleSource>,
```

Comment:
should this be id_to_source? Likewise with the add_ method below

Reply:
oh shit good call 1 sec